Library/Formula/jenkins.rb

```diff
@@ -58,12 +64,12 @@ def caveats; <<-EOS.undent
   test do
     ENV["JENKINS_HOME"] = testpath
     pid = fork do
-      exec "#{bin}/jenkins"
+      exec "java -Djava.io.tmpdir=#{testpath} -jar #{opt_libexec}/jenkins.war"
```

Comment:
> If I pass "-Djava.io.tmpdir=#{HOMEBREW_TEMP}" to bin.write_jar_script(), it applies to running jenkins too. Wouldn't that be preferable?

I'm not sure.  Is the Java tmpdir set to a reasonable default and/or should users expect to be able to override it?